### PR TITLE
Switching all emails to `@electronjs.org` domain

### PR DESCRIPTION
**Pull request by groundwater**
CoC should go to `coc@electronjs.org`
Security should go to `security@electronjs.org`

### CODE REVIEW

**Comment by groundwater** [comment]
"I AM TROP, THE DESTROYER OF REGRESSIONS"

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -34,7 +34,7 @@ This Code of Conduct applies both within project spaces and in public spaces whe
 
 ## Enforcement
 
-Instances of abusive, harassing, or otherwise unacceptable behavior may be reported by contacting the project team at [electron@github.com](mailto:electron@github.com). All complaints will be reviewed and investigated and will result in a response that is deemed necessary and appropriate to the circumstances. The project team is obligated to maintain confidentiality with regard to the reporter of an incident. Further details of specific enforcement policies may be posted separately.
+Instances of abusive, harassing, or otherwise unacceptable behavior may be reported by contacting the project team at [coc@electronjs.org](mailto:coc@electronjs.org). All complaints will be reviewed and investigated and will result in a response that is deemed necessary and appropriate to the circumstances. The project team is obligated to maintain confidentiality with regard to the reporter of an incident. Further details of specific enforcement policies may be posted separately.
```
Comment (waleed-ww) [inline CODE_OF_CONDUCT.md:37]: 37

**Review by Only1adhd228 — COMMENTED** [review]
submit commit

**Comment by Natzakub** [comment]
> CoC should go to `coc@electronjs.org` Security should go to `security@electronjs.org`

Natzakab©®™

**Pull request merged**